locale/en.yml

```diff
@@ -104,3 +104,4 @@ core:
   welcome_user: "Welcome, {username}!"
   write_a_post: Write a Post...
   write_a_reply: Write a Reply...
+  and:  and 
```

Comment:
Is there a trailing space after the "and"? I'm not sure that works or is a good idea; as it's so easy to omit or forget...